static/app/views/organizationGroupDetails/groupActivityItem.spec.jsx

```diff
@@ -0,0 +1,46 @@
+import {render} from 'sentry-test/reactTestingLibrary';
+
+import PullRequestLink from 'sentry/components/pullRequestLink';
+import {GroupActivityType} from 'sentry/types';
+import GroupActivityItem from 'sentry/views/organizationGroupDetails/groupActivityItem';
+
+describe('GroupActivityItem', () => {
+  it('shows correct message for pull request', async function () {
+    const repository = TestStubs.Repository();
+
+    const pullRequest = TestStubs.PullRequest({message: 'Fixes ISSUE-1'});
+
+    const component = render(
+      <GroupActivityItem
+        activity={{
```

Comment:
a ton of these tests exist in `static/app/views/organizationGroupDetails/groupActivity.spec.jsx` it might not make the most sense to test it higher but i think its easier if we keep them all together

Reply:
Thank you, I have moved the test to `static/app/views/organizationGroupDetails/groupActivity.spec.jsx`